.travis.yml

```diff
@@ -3,8 +3,9 @@ language: python
 env:
 - TOX_ENV=py26
 - TOX_ENV=py27
-- TOX_ENV=py33
-- TOX_ENV=py34
+# TODO: re-enable when bravado-core supports py3.x
+#- TOX_ENV=py33
+#- TOX_ENV=py34
```

Comment:
I actually did a timeboxed task of porting bravado_core to py3.x.  Turned out to be a lot more work than I expected so I plan to pick that branch up later. Issues arose mostly because of things that don't exist in py3.x like the `long` type if I remember correctly.

https://github.com/Yelp/bravado-core/compare/master...analogue:py3x_support?expand=1


Ticketed @ https://github.com/Yelp/bravado-core/issues/10